### PR TITLE
[TU] Submission of ObjeID 10300 on behalf of TU

### PR DESCRIPTION
**Pull request by jpradocueva**
The Object inside PR #128 has been recreated here due to the problems with the DDF.xml in the PR.

new pull request template

### CODE REVIEW

File: 10300.xml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+      <Item ID="4000">
+        <Name>ObjectInstanceHandle</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Objlnk</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units></Units>
+        <Description>
+          <![CDATA[The object link is used to refer an Instance of a given Object. An Object link value is composed of two concatenated 16-bits unsigned integers following the Network Byte Order convention. The Most Significant Halfword is an ObjectID, the Least Significant Hafword is an ObjectInstance ID.An Object Link referencing no Object Instance will contain the concatenation of 2 MAX-ID values (null link. Test 5]]>
```
Comment (akeranen) [inline 10300.xml:77]: Seems that we need to fix the description of this RR
>(null link. Test 5

**Review by akeranen — APPROVED** [review]
LGTM. The (already existing) RR 4000 needs fix in the description.

File: 10300.xml (hunk 1 above)
Comment (seanmcilroy29) [reply 10300.xml:77]: also in the description, the word ```Hafword``` should be updated ```Halfword```

File: DDF.xml [hunk 2]
```diff
@@ -2896,12 +2896,14 @@ These LwM2M Object Resources MUST only be changed by a LwM2M Bootstrap-Server or
...
         <ObjectID>10300</ObjectID>
         <URN>urn:oma:lwm2m:x:10300</URN>
-        <Name>Reserved</Name>
-        <Description>TBC</Description>
-        <Owner>TBC</Owner>
+        <Name>LWM2M Meta Object</Name>
+        <Description>Use this meta object to provide one or more locations hosting XML data
+ 	  model definitions for reusable or private and vendor-specific devices,
+ 	  such as those in the range 26241-42768.</Description>
+        <Owner>TU</Owner>
```
Comment (bsilverajan) [inline DDF.xml]: Please amend the name of the Owner as TAU  (which is the official abbreviation of Tampere University), Thank you

File: 10300.xml (hunk 1 above)
Comment (bsilverajan) [reply 10300.xml:77]: The description was copied ad-verbatim from line 21 of Common.xml, which is the description of RR 4000. If I need to take action and modify the description for ObjectID 10300, please let me know. 

As I understand, two amendments are needed: "Hafword" should be corrected to "Halfword", and "(null link. Test 5" should be omitted completely.

Comment (seanmcilroy29) [reply 10300.xml:77]: This update will be reviewed and updated by the Working Group

**Pull request merged**